bezier-rs/docs/interactive-docs/src/components/BezierDrawing.ts

```diff
@@ -51,61 +52,56 @@ class BezierDrawing {
 		this.canvas.addEventListener("mousemove", this.mouseMoveHandler.bind(this));
 		this.canvas.addEventListener("mouseup", this.deselectPointHandler.bind(this));
 		this.canvas.addEventListener("mouseout", this.deselectPointHandler.bind(this));
-		this.ctx.strokeRect(0, 0, this.canvas.width, this.canvas.height);
 		this.updateBezier();
 	}
 
+	clearFigure(): void {
+		this.ctx.clearRect(0, 0, this.canvas.width, this.canvas.height);
+	}
+
 	mouseMoveHandler(evt: MouseEvent): void {
 		const mx = evt.offsetX;
 		const my = evt.offsetY;
 
-		if (
-			this.dragIndex != null &&
-			mx - this.points[this.dragIndex].r > 0 &&
-			my - this.points[this.dragIndex].r > 0 &&
-			mx + this.points[this.dragIndex].r < this.canvas.width &&
-			my + this.points[this.dragIndex].r < this.canvas.height
-		) {
+		if (this.dragIndex !== null && mx - 5 > 0 && my - 5 > 0 && mx + 5 < this.canvas.width && my + 5 < this.canvas.height) {
 			const selectedPoint = this.points[this.dragIndex];
 			selectedPoint.x = mx;
 			selectedPoint.y = my;
 			this.bezier[selectedPoint.mutator](selectedPoint.x, selectedPoint.y);
-
-			this.ctx.clearRect(1, 1, this.canvas.width - 2, this.canvas.height - 2);
-			this.updateBezier();
+			this.clearFigure();
 		}
+		this.updateBezier();
 	}
 
 	mouseDownHandler(evt: MouseEvent): void {
 		const mx = evt.offsetX;
 		const my = evt.offsetY;
 		for (let i = 0; i < this.points.length; i += 1) {
 			if (
-				Math.abs(mx - this.points[i].x) < this.points[i].r + 3 &&
-				Math.abs(my - this.points[i].y) < this.points[i].r + 3 // Fudge factor makes the points easier to grab
+				Math.abs(mx - this.points[i].x) < 5 + 3 &&
+				Math.abs(my - this.points[i].y) < 5 + 3 // Fudge factor makes the points easier to grab
```

Comment:
And I believe the `5` is used here too. And what's the `3`? That should preferably be a const also, or at least a comment explaining why it's 3.